README.md

```diff
@@ -1 +1,69 @@
-# ah-frontend-kronos
+
+Authors Haven - A Social platform for the creative at heart.
+=======
+
+## Vision
+Create a community of like minded authors to foster inspiration and innovation
+by leveraging the modern web.
+
+<!-- TABLE OF CONTENTS -->
+## Table of Contents
+
+* [About Authors Haven](#about-authors-haven)
+  * [Built With](#built-with)
+* [Getting Started](#getting-started)
+  * [Prerequisites](#prerequisites)
+  * [Installation](#installation)
+* [Usage](#usage)
+* [Testing](#testing)
+* [Demo Link](#demo-link)
+
+<!-- ABOUT THE PROJECT -->
+## About Authors Haven
+
+This is a practice project to learn and practice React library in making of a frontend. Find the backend repo [here](https://github.com/andela/ah-backend-kronos). Authors Haven is a tool for bloggers and readers to share and find their creative written content.
+
+### Built With
```

Comment:
@Manorlds-Eaglespark, Could you please try to follow the linting conventions of spaces after every title and lists. helpful resource - https://github.com/DavidAnson/vscode-markdownlint#configure
Thank you.
<img width="1440" alt="Screenshot 2019-07-03 at 15 04 47" src="https://user-images.githubusercontent.com/50827537/60590555-f0b37180-9da4-11e9-996e-bd336fd07e11.png">